cmd/software/deployment.go

```diff
@@ -143,7 +143,7 @@ func newDeploymentCreateCmd(out io.Writer) *cobra.Command {
 	}
 
 	if triggererEnabled {
-		cmd.Flags().IntVarP(&createTriggererReplicas, "triggerer-replicas", "", 1, "Number of replicas to use for triggerer airflow component, valid 0-2")
+		cmd.Flags().IntVarP(&createTriggererReplicas, "triggerer-replicas", "", -1, "Number of replicas to use for triggerer airflow component, valid 0-2")
```

Comment:
The default value set here is also added in the help function, which could mislead users to believe that by default triggerer won't get created. A cleaner approach to do the same thing would be to use keep default here as `1` and then have the logic to check if the a value has been passed by the user or not in `deploymentCreate` function.

The logic could look like this:
```
if !cmd.Flags().Changed("triggerer-replicas") {
        createTriggererReplicas = -1
}
```

Reference: https://pkg.go.dev/github.com/spf13/pflag#FlagSet.Changed